packages/lodestar/src/network/gossip/encoding.ts

```diff
@@ -74,6 +80,9 @@ export class DataTransformSnappy {
    * transformed data will then be used to create a `RawGossipsubMessage` to be sent to peers.
    */
   outboundTransform(topicStr: string, data: Uint8Array): Uint8Array {
+    if (data.length > this.maxSizePerMessage) {
+      throw Error(`ssz_snappy encoded data length ${length} > ${this.maxSizePerMessage}`);
+    }
```

Comment:
Do we really need to limit outbound size? Do other implementations do it?

Reply:
EDIT: We can keep it, since those messages would be rejected by our peers anyway